src/dialects/abstract/connection-manager.js

```diff
@@ -301,11 +301,21 @@ class ConnectionManager {
    *
    * @returns {Promise}
    */
-  async releaseConnection(connection) {
+  releaseConnection(connection) {
     this.pool.release(connection);
```

Comment:
This is sync?

Reply:
It did not await nor return anything, and looking into sequelize-pool, it's doing anything async, just adding the connection back to the available connection set.

Reply:
Strange hm? But I agree looking at async fun without return or await certainly doesn't make sense :)